index.html

```diff
@@ -1,57 +1,57 @@
-                        <p>Bring a pot of salted water to boil. Add the pasta and cook until it is al dente. Reserve 1 cup of pasta water  and then drain the pasta.</p>
-                        <p>In a small  bowl, combine the parsley and lemon juice and season of salt.</p>
-                        <p>Add the pasta and reserved pasta water to the skillet with tomato-onion mixture. Cook until sauce evenly coats the pasta.</p>
-                        <p>Divide the pasta among four bowls. Top with the parsley and smoked farmer cheese.</p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </body>
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+        <meta charset="UTF-8">
+        <title>Recipe</title>
+    </head>
+    <body>
+        <div class="main-container">
+            <div class="left-section">
+                <div class="title-container">
+                    <div class="title">
+                        <h2>Caramelized red onion pasta</h2>
+                    </div>
+                    <div class="title-text">
+                        If you have never had pasta with caramelized onions, now is the time to try it. My favourite variation includes panfried tomato paste, sun-dried tomatoes, and a dash of soy sauce - big sources of umami flavour. Top the pasta with parsley tossed in lemon juice and smoked farmer cheese, to give it even more flavor and zing!
+                    </div>
+                </div>
+                <div class="ingredients-container">
+                    <div class="ingredients">
+                        <h3>Ingredients</h3>
+                        <img src="vegetarian-icon.svg" alt="vege-icon">
+                    </div>
+                    <div class="ingredients-text">
+                        <p>60 ml olive oil</p>
+                        <p>3 medium red onions, thinly sliced</p>
+                        <p>4 garlic cloves, thinly sliced</p>
+                        <p>1/2 teaspoon fine sea salt, plus more for seasoning</p>
+                        <p>1/2 teaspoon red pepper flakes</p>
+                        <p>3 tablespoons tomato paste</p>
+                        <p>50 g sun-dried tomatoes in oil, chopped</p>
+                        <p>2 tablespoons soy sauce</p>
+                        <p>400 g spaghetti, bucatini or bavette</p>
+                        <p>30 g chopped parsley</p>
+                        <p>Juice of 1/2 lemon</p>
+                        <p>60 g smoked farmer cheese</p>
+                    </div>
+                </div>
+            </div>
+            <div class="right-section">
+                <div class="preparation-container">
+                    <div class="preparation">

```

Comment:
You don't need this container

Reply:
I mean ` <div class="preparation">`, The comment points to the 43rd line

Reply:
removed